appcenter-data/Data.js

```diff
@@ -19,13 +19,43 @@ const DefaultPartitions = {
 const Data = {
     TimeToLive,
     DefaultPartitions,
+    setTokenExchangeUrl,
+    isEnabled,
+    setEnabled,
     read,
     list,
     create,
     remove,
     replace
 };
 
+/**
+ * Change the URL used to retrieve CosmosDB resource tokens.
+ *
+ * @param {string} tokenExchangeUrl - Token Exchange service URL.
+ */
+function setTokenExchangeUrl(tokenExchangeUrl) {
```

Comment:
We need to call this method before start (and thus in native code) and it's only to not use prod servers.
Thus I don't think it's useful to expose this in Javascript.

Reply:
I removed `setTokenExchangedUrl`